### PR TITLE
ci: add alpha workflow

### PR DESCRIPTION
**Pull request by yuberdysheva**

### CODE REVIEW

File: .github/workflows/release-alpha.yml [hunk 1]
```diff
@@ -2,20 +2,48 @@ name: Release alpha version
 
 on:
   workflow_dispatch:
+    inputs:
+      manualPackageVersion:
+        type: string
+        required: false
+        description: 'If you have any troubles with release you can set version of package manually, e.g. 3.0.0-alpha.0'
```
Comment (gorgeousvlad) [inline .github/workflows/release-alpha.yml:9]: what kind of troubles could it be?)

File: .github/workflows/release-alpha.yml [hunk 2]
```diff
@@ -2,20 +2,48 @@ name: Release alpha version
...
-      - name: Extract branch name
-        shell: bash
-        run: |
-          echo "branch_name=${GITHUB_REF#refs/heads/}"
-        id: extract_branch
-      - uses: gravity-ui/release-action@v1
+      - run: |
+          if [ "${{ github.event.inputs.manualPackageVersion }}" != "" ]; then
+            if [[ "${{ github.event.inputs.manualPackageVersion }}" == *"alpha"* ]]; then
+              echo "manualPackageVersion is assigned correctly!"
```
Comment (gorgeousvlad) [inline .github/workflows/release-alpha.yml:18]: mb we should only message if name is incorrect?

File: .github/workflows/release-alpha.yml [hunk 3]
```diff
@@ -2,20 +2,48 @@ name: Release alpha version
...
-          npm-dist-tag: alpha
-          default-branch: ${{ env.branch_name }}
+          registry-url: 'https://registry.npmjs.org'
+      - run: npm ci
+        shell: bash
+      - run: npm test
+        shell: bash
+      - name: Bump and commit version
+        run: |
+          git config --local user.email "111915794+gravity-ui-bot@users.noreply.github.com"
```
Comment (gorgeousvlad) [inline .github/workflows/release-alpha.yml:35]: are't we have this email in ENV variables?

File: .github/workflows/release-alpha.yml (hunk 1 above)
Comment (yuberdysheva) [reply .github/workflows/release-alpha.yml:9]: E.g: version already exists

**Review by gorgeousvlad — APPROVED** [review]

**Pull request merged**